### PR TITLE
added fractional knapsack

### LINKED ISSUE #488: adding greedy algorithms in python
**Is your feature request related to a problem? Please describe.**
Hey @rcoder23  I would like to add all major greedy  algorithms like Fractional Knapsack
Huffman Coding, Activity Selection Problem, Egyptian Fraction in python language

**Do you want to work on it**
Yes
I will implement this in python language

Kindly Assign me this under GSSOC 21
Thank you

### PR DESCRIPTION
**Pull request by robin025**
hey @rcoder23
fixes #488 
added fractional knapsack

### CODE REVIEW

File: greedy algorithms/fractional_knapsack.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        else:
+            fractional_val = i
+            table_profit.append(new_list[i][0])
+            table_weight.append(new_list[i][1])
+            ratio_table.append(new_list[i][2])
+
+            break
+
+    if capacity > 0:
+        max_profit += capacity * \
```
Comment (rcoder23) [inline greedy algorithms/fractional_knapsack.py:47]: **\** cause error

File: greedy algorithms/fractional_knapsack.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+
+# Entre Profit Value : 12 8 9 6 14 30
+# Entre weight  Values  : 98 45 32 69 10 3
+# Entre Capacity of the knapsack28
+# Fractional Knapsack
+# Profit :30              Weight  : 3             Profit/Weight : 10.0
+# Profit :14              Weight  : 10            Profit/Weight : 1.4
+# Profit :9               Weight  : 32            Profit/Weight : 0.28125
+# Total Profit : 48.21875
```
Comment (rcoder23) [inline greedy algorithms/fractional_knapsack.py:73]: Use  '''     '''

**Review by rcoder23 — CHANGES_REQUESTED** [review]

File: greedy algorithms/fractional_knapsack.py (hunk 1 above)
Comment (robin025) [reply greedy algorithms/fractional_knapsack.py:47]: @rcoder23 done

File: greedy algorithms/fractional_knapsack.py (hunk 2 above)
Comment (robin025) [reply greedy algorithms/fractional_knapsack.py]: @rcoder23 done

**Comment by robin025** [comment]
@rcoder23  done...solved error

File: greedy algorithms/fractional_knapsack.py (hunk 2 above)
Comment (robin025) [reply greedy algorithms/fractional_knapsack.py]: @rcoder23  done with changes as suggested

**Comment by robin025** [comment]
@rcoder23  done with changes as suggested

File: greedy algorithms/fractional_knapsack.py (hunk 1 above)
Comment (rcoder23) [reply greedy algorithms/fractional_knapsack.py:47]: @robin025 Not resolve still shows error

**Comment by robin025** [comment]
Done With the error @rcoder23

File: greedy algorithms/fractional_knapsack.py (hunk 1 above)
Comment (robin025) [reply greedy algorithms/fractional_knapsack.py]: @rcoder23  I have maded the required changes

**Review by rcoder23 — APPROVED** [review]

**Comment by robin025** [comment]
Hello @rcoder23 ,
I had a discussion with Praveen who is doing all the leaderboard stuff of GSSoC21 and he told all of us that the GSSoC21 tag should look like *GSSoC21* and it's case sensitive, so if the tag doesn't look like this *GSSoC21* then that PR will not be counted, so can you please update the tag in all my PRs thatbhave reference to issue #488 
Note : there are only 4 PRs

Thankyou

**Pull request merged**